README.md

```diff
@@ -116,7 +116,7 @@ import App from '../src/App.svelte'
 import {render} from 'svelte-testing-library'
 describe('App', () => {
   test('should render greeting', () => {
-    const {getByText} = render(App, {props: {name: 'world'}})
+    const {getByText, container} = render(App, {props: {name: 'world'}})
```

Comment:
can you remove this example from here and move it into its own section for containers in the docs website?  https://testing-library.com/docs/svelte-testing-library/intro

Reply:
```suggestion
    const {getByText} = render(App, {props: {name: 'world'}})
```

Reply:
Will do!